README.md

```diff
@@ -194,6 +194,40 @@ brew install docker-completion
 Now you will need to start the Docker service: run Spotlight and type in
 'docker'.
 
+### Setup: PostgreSQL
+
+To run postgres in Docker, first you will need the image:
+
+```console
+docker pull postgres:11.6
```

Comment:
```suggestion
$ docker pull postgres:11.6
```

Reply:
This throws a markdownlint error.

```
README.md:202 MD014/commands-show-output Dollar signs used before commands without showing output [Context: "$ docker pull postgres:11.6"]
```

The docs link [here](https://cirosantilli.com/markdown-style-guide#dollar-signs-in-shell-code) for rationale. I don't really have a strong opinion, but it kinda makes sense I guess, and the error would have to be disabled.

Reply:
Okay, I won't block on it, but I noticed that without the $ the "docker pull postgres:11.6" text is all blue. But I don't have a strong opinion around using the console vs. shell vs. text syntax with the code blocks.

Reply:
That's a fair point. I just discovered the "console" language highlighting thing. I'll try it the other way and see how it looks.